db/migrate/20150813185339_create_subscriptions.rb

```diff
@@ -0,0 +1,73 @@
+class CreateSubscriptions < ActiveRecord::Migration
+  def change
+    create_table "katello_subscriptions" do |t|
+      t.string :product_name
+      t.string :product_id
```

Comment:
At first glance, not seeing why store `product_id` and `product_name`  instead of associating to a product?